package.json

```diff
@@ -42,7 +46,8 @@
     "gulp-zip": "^4.2.0",
     "jquery": "^3.3.1",
     "nunjucks": "^3.1.3",
-    "sass-lint": "^1.12.1"
+    "sass-lint": "^1.12.1",
+    "start-server-and-test": "^1.7.9"
```

Comment:
We can do the same thing without installing another dependency: `"test:cypress": "npm start && cypress",` We don't need to install "start-server-and-test": "^1.7.9" to do this. `npm uninstall start-server-and-test`

Reply:
updated with "test:cypress": "npm start & npm run cypress" and uninstalled start-server-and-test